src/providers/web3.provider.ts

```diff
@@ -188,6 +188,17 @@ export default class Web3Provider {
       .call();
   }
 
+  static async getETHBalance(): Promise<number> {
+    const account = Web3Provider.web3.eth.accounts.privateKeyToAccount(
+      process.env.ACCOUNT_PRIVATE_KEY,
```

Comment:
If account private key is not set, it should not fail and should return 0, which already will mean exceptional because grafana will have emergency